dotcom-rendering/src/components/Elements.amp.tsx

```diff
@@ -311,9 +312,20 @@ export const Elements = (
 						pillar={pillar}
 					/>
 				);
+			case 'model.dotcomrendering.pageElements.TimelineAtomBlockElement':
```

Comment:
Looks like you can do this: 

```suggestion
            case 'model.dotcomrendering.pageElements.TimelineBlockElement':
			case 'model.dotcomrendering.pageElements.TimelineAtomBlockElement':
```

and remove these [lines](https://github.com/guardian/dotcom-rendering/pull/10996/files#diff-12740f4ec6664cfe83fbc21405c9e0756bd6173ddd918b91dab6149df7f7721aR326-R336)

Reply:
Sorry ignore, just read the description 🤦‍♀️